runtime/opt/taupage/init.d/01-register-scalyr-agent.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+import logging
+import subprocess
+import re
+import boto.utils
+import json
+import ast
+
+
+from taupage import get_config
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+scalyr_agent_config_file = '/etc/scalyr-agent-2/agent.json'
+
+main_config = get_config()
+logging_config = main_config.get('logging')
+
+mount_custom_log = main_config.get('mount_custom_log')
+rsyslog_aws_metadata = main_config.get('rsyslog_aws_metadata')
+
+application_log_path = '/var/log/application.log'
+custom_log_path = '/var/log-custom/*.log'
+auth_log_path = '/var/log/auth.log'
+sys_log_path = '/var/log/syslog'
+
+jwt_redaction = [
+    {
+        'match_expression': 'eyJ[a-zA-Z0-9/+_=-]{5,}\\.eyJ[a-zA-Z0-9/+_=-]{5,}\\.[a-zA-Z0-9/+_=-]{5,}',
+        'replacement': '+++JWT_TOKEN_REDACTED+++'
+    }
+]
+
+if rsyslog_aws_metadata:
+    sys_log_parser = 'systemLogMetadata'
+else:
+    sys_log_parser = 'systemLog'
+
+if not logging_config:
+    account_key = main_config.get('scalyr_account_key')
+    region = main_config.get('scalyr_region', 'eu')
+    application_log_parser = main_config.get('scalyr_application_log_parser', 'slf4j')
+    custom_log_parser = main_config.get('scalyr_application_log_parser', 'slf4j')
+    scalyr_agent_enabled = True
+    ship_application_log = True
+    application_log_sampling = None
+    ship_custom_log = True
+    custom_log_sampling = None
+    ship_auth_log = True
+    auth_log_sampling = None
+    ship_sys_log = True
+    sys_log_sampling = None
+else:
+    account_key = logging_config.get('scalyr_account_key')
+    region = logging_config.get('scalyr_region', 'eu')
+    application_log_parser = logging_config.get('scalyr_application_log_parser', 'slf4j')
+    custom_log_parser = logging_config.get('scalyr_application_log_parser', 'slf4j')
+    fluentd_enabled = logging_config.get('fluentd_enabled')
+    scalyr_agent_enabled = logging_config.get('scalyr_agent_enabled', not fluentd_enabled)
+    ship_all_logs = logging_config.get('use_scalyr_agent_all', not fluentd_enabled)
+    ship_application_log = logging_config.get('use_scalyr_agent_applog', ship_all_logs)
+    application_log_sampling = logging_config.get('scalyr_agent_applog_sampling')
+    ship_custom_log = logging_config.get('use_scalyr_agent_customlog', ship_all_logs)
+    custom_log_sampling = logging_config.get('scalyr_agent_customlog_sampling')
+    ship_auth_log = logging_config.get('use_scalyr_agent_authlog', ship_all_logs)
+    auth_log_sampling = logging_config.get('scalyr_agent_authlog_sampling')
+    ship_sys_log = logging_config.get('use_scalyr_agent_syslog', ship_all_logs)
+    sys_log_sampling = logging_config.get('scalyr_agent_syslog_sampling')
+
+if not (scalyr_agent_enabled and account_key):
+    logger.info('Found no Scalyr key or Scalyr agent not enabled; skipping Scalyr Agent init')
+    raise SystemExit()
```

Comment:
`sys.exit()` does same but more readable from my perspective — we're want to `exit`, not to raise & catch exceptions.